.github/workflows/main_lastman-standing-app.yml

```diff
@@ -22,33 +25,35 @@ jobs:
           java-version: '17'
           distribution: 'microsoft'
 
-      - name: Build with Maven
-        run: mvn clean install
+      - name: unit tests

```

Comment:
Im not sure what this is? I thought mvn clean install ran unit tests fine but could be wrong

Reply:
Is this branch also not setup for pipeline yet?

Reply:
This is already in main, I just merged main into my branch to solve conflicts.

Reply:
Ah okay sound, good work will approve, @CaoimhinHawk1 need pipeline to run on this too in future